compiler/il/VectorOperations.enum

```diff
@@ -146,6 +146,70 @@ VECTOR_OPERATION_MACRO(\
    /* .ifCompareOpCode         = */ TR::BadILOp, \
    /* .description             =    store indirect mask */ \
 )
+VECTOR_OPERATION_MACRO(\
+   /* .operation               = */ mTrueCount, \
+   /* .name                    = */ "mTrueCount", \
+   /* .properties1             = */ 0, \
+   /* .properties2             = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE, \
+   /* .properties3             = */ ILProp3::MaskReduction, \
+   /* .properties4             = */ 0, \
+   /* .dataType                = */ TR::Int64, \
+   /* .typeProperties          = */ TR::NoType, \
+   /* .childProperties         = */ ONE_CHILD(ILChildProp::UnspecifiedChildType), \
+   /* .swapChildrenOperation   = */ TR::vBadOperation, \
+   /* .reverseBranchOperation  = */ TR::vBadOperation, \
+   /* .booleanCompareOpCode    = */ TR::BadILOp, \
+   /* .ifCompareOpCode         = */ TR::BadILOp, \
+   /* .description             =    mask true count */ \
+)
+VECTOR_OPERATION_MACRO(\
+   /* .operation               = */ mFirstTrue, \
+   /* .name                    = */ "mFirstTrue", \
+   /* .properties1             = */ 0, \
+   /* .properties2             = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE, \
+   /* .properties3             = */ ILProp3::MaskReduction, \
+   /* .properties4             = */ 0, \
+   /* .dataType                = */ TR::Int64, \
+   /* .typeProperties          = */ TR::NoType, \
+   /* .childProperties         = */ ONE_CHILD(ILChildProp::UnspecifiedChildType), \
+   /* .swapChildrenOperation   = */ TR::vBadOperation, \
+   /* .reverseBranchOperation  = */ TR::vBadOperation, \
+   /* .booleanCompareOpCode    = */ TR::BadILOp, \
+   /* .ifCompareOpCode         = */ TR::BadILOp, \
+   /* .description             =    mask first true */ \
+)
+VECTOR_OPERATION_MACRO(\
+   /* .operation               = */ mLastTrue, \
+   /* .name                    = */ "mLastTrue", \
+   /* .properties1             = */ 0, \
+   /* .properties2             = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE, \
+   /* .properties3             = */ ILProp3::MaskReduction, \
+   /* .properties4             = */ 0, \
+   /* .dataType                = */ TR::Int64, \
+   /* .typeProperties          = */ TR::NoType, \
+   /* .childProperties         = */ ONE_CHILD(ILChildProp::UnspecifiedChildType), \
+   /* .swapChildrenOperation   = */ TR::vBadOperation, \
+   /* .reverseBranchOperation  = */ TR::vBadOperation, \
+   /* .booleanCompareOpCode    = */ TR::BadILOp, \
+   /* .ifCompareOpCode         = */ TR::BadILOp, \
+   /* .description             =    mask last true */ \
+)
+VECTOR_OPERATION_MACRO(\
+   /* .operation               = */ mToLong, \
```

Comment:
I think the name of this opcode will cause confusion with m2l opcode.

Reply:
Renamed to mToLongBits